__tests__/__utils__/assertions.ts

```diff
@@ -106,3 +108,71 @@ export async function assertFailingGraphQLMutation({
   })
   expect(response?.errors?.[0]?.extensions?.code).toEqual(expectedError)
 }
+
+/**
+ * Assertation that a certain error event occured. Since we use Sentry this
+ * function checks that a Sentry event was thrown.
+ *
+ * TODO: This function has not a good error message in case the asseration
+ * fails. I reccomend you to investigate `global.sentryEvents` with
+ * `console.log()` or something similar when your tests fail.
+ *
+ * @example
+ * // assertation that at least one error occured
+ * assertErrorEvent()
```

Comment:
```suggestion
 * @example
 * ```ts
 * // assertion that at least one error occurred
 * assertErrorEvent()
 * ```
```